systems/WikibaseRepo/05-Building_Block_View.md

```diff
@@ -4,9 +4,38 @@
 
 TBA
 
-## External Interfaces
-
-### Action API Edit Entity
+## Action API modules
+
+| Building Block                          | Responsibility                                                                              |
+| --------------------------------------- | ------------------------------------------------------------------------------------------- |
+| wbavailablebadges                       | Query available badge items                                                                 |
```

Comment:
I'm not sure there is a difference between query and get in this table.
Perhaps we should stick to Get throughout?
THis also highlights another tiny thing we could change in real code to bring everything toward consistency.

Reply:
I copy-pasted (and slightly modified) from [the API help page](https://www.wikidata.org/w/api.php?action=help&modules=main) and would love to see our docs in sync with that one.

Reply:
I'm also pro updating the api help page docs, but I think "Get" is correct and "query" is misleading and inconsistent with other descriptions in this table.
We should say what we mean in these docs so as not to confuse others that look at them, and if that means straying slightly from sources we can copy from I think thats okay.
Also, lets go make the gerrit change!

Reply:
not sure where exactly to fix this, is it [here](https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/extensions/Wikibase/+/refs/heads/master/repo/i18n/api/en.json#30)?

Reply:
Yes, and I also see an inconsistency with the wording in the first example `apihelp-wbavailablebadges-example-1`